simapp/sim_test.go

```diff
@@ -218,11 +218,21 @@ func TestAppImportExport(t *testing.T) {
 		storeA := ctxA.KVStore(skp.A)
 		storeB := ctxB.KVStore(skp.B)
 
-		failedKVAs, failedKVBs := sdk.DiffKVStores(storeA, storeB, skp.Prefixes)
+		failedKVAs, failedKVBs := storetypes.DiffKVStores(storeA, storeB, skp.Prefixes)
```

Comment:
the main issue here is a public method is returning an internal type, im not sure how to best approach fixing the ugliness below. Any advice @amaurym @alexanderbez

Reply:
Yeah that's def a codesmell! But `DiffKVStores` returns `[]kv.Pair` which is public. What internal type are you referring to?

Reply:
DiffKVStores returns a private type of kv. I can make it public but trying to avoid it

Reply:
Ohh you meant _internal_? I see. Let's not conflate the two terms. I was confused. Looks like you got it resolved though! wohoo